conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
```

Comment:
I think this line is a bit long. It doesn't look like it's triggering a markdown lint error but it would be nice to break it up on multiple lines.

Reply:
I personally don't think it needs to be broken up more, I think it's fine as one thought.